### PR TITLE
Transaction requests

### PR DESCRIPTION
**Pull request by TalPat**
Transaction requests are validated and an asynchronous response is sent to the switch

### CODE REVIEW

File: packages/backend/migrations/20191202155236_create_mojatransaction_request_table.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('mojatransaction_request', function (table) {
```
Comment (matdehaast) [inline packages/backend/migrations/20191202155236_create_mojatransaction_request_table.js]: Lets rather us camelCase, it much easier to stay consistent in our code

```
mojaTransactionRequest
```

File: packages/backend/migrations/20191202155236_create_mojatransaction_request_table.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+       table.uuid('transactionRequestId')
+       table.string('serializedRequest')
+       table.boolean('valid')
+    })
+  
+};
+
+exports.down = function(knex) {
+  return knex.schema
+      .dropTableIfExists("mojatransaction_request")
```
Comment (matdehaast) [inline packages/backend/migrations/20191202155236_create_mojatransaction_request_table.js]: same as above

File: packages/backend/src/apis/requestResponses.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+export type TransactionRequestError = {
+  errorInformation: {
+    errorCode: string;
+    errorDescription: string;
+    extensionList: ExtensionList;
+  }
+}
+
+export interface RequestResponses {
```
Comment (matdehaast) [inline packages/backend/src/apis/requestResponses.ts]: Very generic name when our codebase could have lots of different business concerns. Also looks very much like a service. Rather put in /services directory and call `MojaResponseService`.

File: packages/backend/src/apis/requestResponses.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import axios, { AxiosResponse } from 'axios'
+import { ExtensionList } from '../services/transaction-request-service'
+
+const putBase: string = process.env.PUT_BASE_URI || 'http://localhost:8008' // base uri for testing
```
Comment (matdehaast) [inline packages/backend/src/apis/requestResponses.ts]: Change to `baseMojaUrl`

**Comment by TalPat** [comment]
Requested changes have been made.

**Pull request merged**